packages/odata-service-inquirer/src/prompts/prompts.ts

```diff
@@ -53,7 +53,6 @@ function getDatasourceTypeQuestion(options?: DatasourceTypePromptOptions): YUIQu
                 [
                     DatasourceType.businessHub,
```

Comment:
Doesn't have to be in this PR but `businessHub` can eventually go

Reply:
Thx @cianmSAP , yes, I was waiting for confirmation before I remove it completely.